src/System.Private.ServiceModel/tests/Scenarios/Contract/Data/Contract.Data.Tests.csproj

```diff
@@ -17,7 +17,7 @@
     <ProjectGuid>{4361D851-B56C-44CD-B7BD-D9EFD32F94AD}</ProjectGuid>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="DataContractTests.cs" />
+    <Compile Include=".\*.cs" />
```

Comment:
Nit: we normally use "**\*.cs" just in case someone adds subfolders.  Not required for checkin -- we can update this if anyone does this.